contentctl/objects/abstract_security_content_objects/detection_abstract.py

```diff
@@ -322,12 +322,13 @@ def nes_fields(self) -> Optional[str]:
     @property
     def providing_technologies(self) -> List[ProvidingTechnology]:
         return ProvidingTechnology.getProvidingTechFromSearch(self.search)
-        
-    
+
+    # TODO (#247): Refactor the risk property of detection_abstract
     @computed_field
     @property
     def risk(self) -> list[dict[str, Any]]:
         risk_objects: list[dict[str, str | int]] = []
+        # TODO (#246): "User Name" type should map to a "user" risk object and not "other"
```

Comment:
Why not make this change now instead of tracking it as a TODO?


Second, (I realize you didn't touch these values, but still think this is a valuable comment for the whole team):
We have lots of "magic enums/lists/values" that have come from... somewhere. We should link to documentation when possible so we have a record of why they were chosen (and can update them, if required).  These are all over the place in contentctl - if there is public documentation on where this list of types came from, we should:
1. include it as a comment
2. Break it out into an enum or class

Reply:
Will make that change if you agree then!

Also +1 on the magic enums. We also don't use the enums consistently. For example, we use the observable type enums in validation, but we have a separate static list that we use in risk generation. Unifying those is a good idea

Reply:
Actually, addressing this here feels a bit like scope creep as it is not part of the feature I am delivering. I also believe @ljstella may already be tracking this change?

Reply:
Agreed as out of scope. Let's lean on the @ljstella updates for this.